Ntuplizer/interface/FunctionLibrary.h

```diff
@@ -126,6 +126,21 @@ namespace
 
                                  return out;
                                });
+        addTo["lheWeights"] =
+          std::function<FType>([](const edm::Ptr<T>& obj, uwvv::EventInfo& evt, const std::string& option)
+                               {
+                                 std::vector<float> out;
+                                 
+                                 if (!evt.lheEventInfo().isValid()) 
+                                   {
+                                     out.push_back(-1);
+                                     return out;
```

Comment:
I'd probably prefer to just throw an exception in this case. Maybe it's holding the user on too tight a leash, but if you're running this on data you likely have other problems...